<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ToxicMusicPlayer</class>
 <widget class="QMainWindow" name="ToxicMusicPlayer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>802</width>
    <height>610</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ToxicMusicPlayer</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-image: url(source/image/backForForm.jpg);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>370</y>
      <width>791</width>
      <height>191</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="0" rowspan="2">
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="2" column="1" colspan="4">
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="5" column="0" colspan="3">
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="2">
        <widget class="QPushButton" name="playNext">
         <property name="styleSheet">
          <string notr="true">background-image: url(source/image/backButton.jpg);
color: rgb(0, 255, 255);
</string>
         </property>
         <property name="text">
          <string>&gt;&gt;</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QPushButton" name="Play_Pause">
         <property name="styleSheet">
          <string notr="true">background-image: url(source/image/backButton.jpg);
color: rgb(0, 255, 255);</string>
         </property>
         <property name="text">
          <string>Play</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QPushButton" name="playBack">
         <property name="styleSheet">
          <string notr="true">background-image: url(source/image/backButton.jpg);
color: rgb(0, 255, 255);</string>
         </property>
         <property name="text">
          <string>&lt;&lt;</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0" colspan="3">
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="2" column="1">
        <widget class="QPushButton" name="playStop">
         <property name="styleSheet">
          <string notr="true">background-image: url(source/image/backButton.jpg);
color: rgb(0, 255, 255);</string>
         </property>
         <property name="text">
          <string>Stop</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="timeAfterStart">
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">ackground-image: url(source/image/backButton.jpg);
color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>0:00</string>
       </property>
      </widget>
     </item>
     <item row="0" column="5" rowspan="3">
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="1" column="2">
      <widget class="QSlider" name="timeSlider">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="0" column="2" colspan="2">
      <widget class="QLineEdit" name="lineEdit">
       <property name="text">
        <string>Now playing: none</string>
       </property>
      </widget>
     </item>
     <item row="1" column="4">
      <widget class="QLabel" name="songTime">
       <property name="styleSheet">
        <string notr="true">ackground-image: url(source/image/backButton.jpg);
color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>0:00</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0" colspan="2">
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="1" column="6">
      <widget class="QSlider" name="volumeSlider">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>0</y>
      <width>291</width>
      <height>241</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">ackground-image: url(source/image/music.jpeg);</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="AddMusic">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>270</y>
      <width>301</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_WriteNameorUrl">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>250</y>
      <width>181</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Write the name or URL</string>
    </property>
   </widget>
   <widget class="QPushButton" name="AddMusicButton">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>280</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Add to queue</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>802</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
